infra.bs

```diff
@@ -630,6 +630,13 @@ out-of-bounds, except when used with <a for=list>exists</a>.
 "<code>b</code>", "<code>c</code>", "<code>a</code>" ». Then |example|[1] is the <a>string</a>
 "<code>b</code>".
 
+<p>Additionally, a literal syntax can be used to express a <a>list</a> of integers: the range
+<var>n</var> to <var>m</var>, inclusive, creates a new <a>list</a> whose <a for=list>items</a> are
+all integers in the range <var>n</var> to <var>m</var>, inclusive.
```

Comment:
I agree about circular though, but the alternative is to define it in terms of some kind of loop, which also seems somewhat bad.